src/main/groovy/edu/oregonstate/mist/trailsapi/resources/TrailResources.groovy

```diff
@@ -81,4 +81,25 @@ public class TrailsResource extends Resource {
 	Boolean trailValidator(Trail trail) {
 		trail.name && trail.zipCode && trail.difficulty
 	}
+
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
+	Response getTrail (@QueryParam("name") Optional<String> name,
+					@QueryParam("difficulty") Optional<String> difficulty,
```

Comment:
Could you align all these query parameters with the first one in this line? `Response getTrail (@QueryParam("name") Optional<String> name,`